diesel/src/pg/types/ranges.rs

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
Can you run this file through `rustfmt` when you're done? (Latest version default settings)

I'm planning on doing it to the whole codebase soon, one less place that needs to churn.